time start_time
time end_time
time speech_duration
time received_time
string transcript
float64 confidence